submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/MongoDbConfig.kt

```diff
@@ -28,7 +28,7 @@ const val CHANGE_LOG_LOCK = "submitter_mongockLock"
 @Configuration
 @EnableMongoRepositories(
     basePackageClasses = [
-        SubmissionMongoRepository::class,
```

Comment:
Why are we removing this one? 🤔

Reply:
this repository is not any more a normal one but reactive one so this one is loaded true reactive mongo config